tueplots/figsizes.py

```diff
@@ -154,7 +154,7 @@ def jmlr2001(
     )
 
 
-def neurips2021(
+def neurips_defaults(
```

Comment:
How do you feel about renaming this function to `_neurips_common` and moving it to below the functions where it is called? The underscore would go more in line with other helper functions (e.g., `_from_base_in`), `defaults` sounds a bit like a default argument instead of shared parameters, and the position underneath the other functions makes it easier to see which function calls which other function.

What do you think? 

# 
P.S. sorry for doing <a href="https://aws1.discourse-cdn.com/github/original/2X/7/764bbe98d96fbd527c5e084fb903bf20bfb83967.png">this</a>.